translate/convert/test_po2json.py

```diff
@@ -0,0 +1,26 @@
+# -*- coding: utf-8 -*-
+from translate.convert import po2json
+from translate.misc import wStringIO
+from translate.storage import po
+
+
+class TestPO2JSON(object):
+
+    def po2json(self, po_source, json_template):
+        """helper that converts po source to json source without requiring files"""
+        input_file = wStringIO.StringIO(po_source)
+        json_file = wStringIO.StringIO(json_template)
+        input_po = po.pofile(input_file)
+        convertor = po2json.rejson(json_template, input_po)
+        output_json = convertor.convertstore()
+        return output_json
+
+    def test_basic(self):
+        """test a basic po to json conversion"""
+        json_source = '''{ "text": "A simple string"}'''
```

Comment:
I would prefer this to be renamed to `json_template` since it states better its purpose.